local_sensors_compose.yaml

```
@@ -1,5 +1,15 @@
 services:
 
+  av_bms:
+    image: "ghcr.io/ipab-rad/av_bms:latest"
```

Beacuse this is from `local_sensor_compose` this line should be:
```suggestion
    image: "av_bms:latest"
```